signatures/example_signatures.json

```diff
@@ -0,0 +1,90 @@
+{
+    "ipSignature": [
+        "^176\\.191\\.129\\.\\d{1,3}$",
```

Comment:
Currently, inside web_ids the ip-field inside ipSignatures-objects is used. Like it was in the original signature file.
This might even be better than having an array (as mentioned in #2)
Keeping the ipSignature as object like before, gives us the option to provide a type field to define, if the ip match is of type INFO, WARNING, ...
Since we only parse fields known to us, one could for example add a msg-field to provide additional information for a certain match.

So in summary:
I would keep the ipSignature in the form

~~~
{
   "ip" : "<some ip to match>",
   "type" : "<INFO | WARNING | ...>,
   "msg" : "Optional message field that we ignore"
}
~~~

Reply:
Ok. Will change this

Reply:
@mhatzl Is it ok to just include an IP key in the object in the example file and the documentation for now? Including the other keys we have not yet implemented would be bad programming practice.

Reply:
Yes thats good